core/src/main/java/de/danoeh/antennapod/core/util/playback/IPlayer.java

```diff
@@ -6,34 +6,24 @@
 import java.io.IOException;
 
 public interface IPlayer {
```

Comment:
are these not needed anymore?

Reply:
Intellij told me they are unused. Additionally, every class that implemented them was just throwing RuntimeExceptions in those methods (see diff). So I just removed them (ExoPlayer does not seem to support those things anyway)